src/com/dabomstew/pkrandom/Settings.java

```diff
@@ -49,7 +49,7 @@
 
     public static final int VERSION = Version.VERSION;
 
-    public static final int LENGTH_OF_SETTINGS_DATA = 48;
+    public static final int LENGTH_OF_SETTINGS_DATA = 49;

```

Comment:
I actually originally had the TMs setting here but I wasn't sure if it would be appropriate so I moved it to the end. At any rate, I went ahead and moved the TM and tutor evolution settings to byte 19 and reduced the settings length back to 48.